src/aelplermagronen.md

```diff
@@ -1,22 +1,25 @@
 # Älplermagronen (Alpine macaroni)
 
+![Älplermagronen](pix/aelplermagronen.webp)
+
 A swiss favorite, _Älplermagronen_ combines pretty much everything you have at your disposal in your alpine chalet.
 It's the definition of comfort food for the Swiss.
 
-Serves 5-6.
-
-Cooking time: ~30 minutes
+- 🍳 Cook time: ~30 minutes
+- 🍽️  Servings: 4
 
 ## Ingredients
 
 - ~150g (1/3 lb) bacon cubes
 - 3 onions (medium size)
 - 400g (15 oz) potatoes (firm/waxy)
-- 2L (1/2 gal) milk
-- 200g (7oz) macaroni (dry weight)
+- 1 - 2L (1/4 - 1/2 gal) milk
+- 400g (7oz) macaroni (dry weight)
```

Comment:
400g is not 7oz, so this should be updated. Not sure if you wanted a range like you did above or not though

Reply:
Good catch! Thanks for that.